modules/openapi-generator/pom.xml

```diff
@@ -258,7 +258,7 @@
         <diffutils-version>1.3.0</diffutils-version>
         <guava-version>26.0-jre</guava-version>
         <generex-version>1.0.2</generex-version>
-        <rxgen-version>1.1</rxgen-version>
+        <rxgen-version>1.3-SNAPSHOT</rxgen-version>
```

Comment:
Don't think we can use SNAPSHOT version in the upcoming v5.0.0 major release.

Reply:
In that case, shall I delete this PR. Because 1.3 is not yet released.

Reply:
What about upgrading to 1.2? Would that address the issue you reported?

Reply:
No

Reply:
Please still upgrade it to 1.2 for the time being before the 1.3 release.

Reply:
done

Reply:
The maintainer of Rxgen may be able to put out a non-snapshot release before our major release. Should we wait until ~3 days before our release to see if he releases?

Reply:
Sure we can wait.

Reply:
Hello @spacether, @wing328 and @code-lucidal58.

The RgxGen version 1.3 is released and available in maven central.

Reply:
Thank you!